pennylane/qchem/openfermion_obs.py

```diff
@@ -881,22 +857,59 @@ def molecular_hamiltonian(
             corresponding to the qubit number equal to its index.
             For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted for
             partial mapping. If None, will use identity map.
-        alpha (array[float]): exponents of the primitive Gaussian functions
-        coeff (array[float]): coefficients of the contracted Gaussian functions
         args (array[array[float]]): initial values of the differentiable parameters
         load_data (bool): flag to load data from the basis-set-exchange library
         convert_tol (float): Tolerance in `machine epsilon <https://numpy.org/doc/stable/reference/generated/numpy.real_if_close.html>`_
             for the imaginary part of the Hamiltonian coefficients created by openfermion.
             Coefficients with imaginary part less than 2.22e-16*tol are considered to be real.
 
+
+    .. details::
+        :title: Usage Details
+
+        Old interface used ``molecular_hamiltonian`` function with molecular information as separate arguments:
+
+        .. py:function:: molecular_hamiltonian(symbols, coordinates, name="molecule", charge=0, mult=1, basis="sto-3g", method="dhf",
+        active_electrons=None, active_orbitals=None, mapping="jordan_wigner", outpath=".", wires=None, alpha=None,
+        coeff=None, args=None, load_data=False, convert_tol=1e012)
```

Comment:
```suggestion
        .. py:function:: molecular_hamiltonian(symbols, coordinates, name="molecule", charge=0, mult=1, basis="sto-3g", method="dhf", active_electrons=None, active_orbitals=None, mapping="jordan_wigner", outpath=".", wires=None, alpha=None, coeff=None, args=None, load_data=False, convert_tol=1e012)
```

I know it looks weird, but it needs to be in a single line for it to render properly on the website.